### DISCUSSION TITLE
I wanted to know the front direction of the quadcopter and the direction of the LEDs

### DISCUSSION THREAD

**Discussion by anish-natekar** [jyjblrd/Mocap-Drones#24] [Q&A] [answered]
I wanted to know in the quadcopter which direction is the front/forward direction so that I can know if I need to put one of the LEDs on the right or left side of the quadcopter (The left and right considering looking at the TOP DOWN view with front of the Quadcopter facing forwards/up). I have an open-source sub 100g custom brushed Wifi quadcopter I made last summer which I want to control using this motion capture system (Link to PicoW_Copter project - https://github.com/anish-natekar/PicoW_Copter).

**Comment by jyjblrd** [comment, marked as answer]
The code to detect markers in a triangle formation (ie. locate the drone from three markers) is [here](https://github.com/jyjblrd/Mocap-Drones/blob/8d5169e466d6ece850094e5713f7ec2fbbe4cff0/computer_code/api/helpers.py#L424). In my example I used two drones, one with the led on the left and one with the led on the right. Really it is quite arbitrary, and you can easily change the code that finds the marker formations you put on the drone.
